tests/functional/aws-node-sdk/test/legacy/authV2QueryTests.js

```diff
@@ -40,7 +40,7 @@ describe('aws-node-sdk v2auth query tests', function testSuite() {
     // AWS allows an expiry further in the future
     itSkipAWS('should return an error code if expires header is too far ' +
         'in the future', done => {
-        const params = { Bucket: bucket, Expires: 3605 };
+        const params = { Bucket: bucket, Expires: 100001 };
```

Comment:
Please add a comment what this time represents and what our time limit is so we remember.